backend/src/dao/TeamEventsDao.ts

```diff
@@ -79,7 +79,7 @@ export default class TeamEventsDao {
   /* Returns all team event information details through 'TeamEventInfo' objects */
   static async getAllTeamEventInfo(): Promise<TeamEventInfo[]> {
     const docRefs = await teamEventsCollection
-      .select('name', 'date', 'numCredits', 'hasHours', 'uuid', 'isInitiativeEvent')
+      .select('name', 'date', 'numCredits', 'hasHours', 'uuid', 'isInitiativeEvent', 'maxCredits')
```

Comment:
I think at this point all the fields are just being selected right? I think the original motivation for selecting specific fields isn't an issue anymore